libs/designsystem/src/lib/components/modal/services/modal-navigation.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@angular/core';
-import { ActivatedRoute, NavigationEnd, Route, Router, Routes } from '@angular/router';
+import { ActivatedRoute, NavigationEnd, Params, Route, Router, Routes } from '@angular/router';
 import { EMPTY, Observable } from 'rxjs';
-import { filter, first, map, pairwise, startWith } from 'rxjs/operators';
+import { filter, first, map, pairwise, startWith, tap } from 'rxjs/operators';
```

Comment:
Remove unused import
```suggestion
import { filter, first, map, pairwise, startWith } from 'rxjs/operators';
```